### PR TITLE
Portal nav post login

### PR DESCRIPTION
**Pull request by dorianjp**
# Summary | Résumé

Trello
https://trello.com/c/gPhP8j6T/800-portal-navigation-after-log-in

New landing page for those with alert permission
Moved some items from nav to footer (support and about)
Added home link to nav
New tests

# Test instructions | Instructions pour tester la modification

* Login without alert permission, the landing page should be the same OTK page
* Login with alert permission, the landing page should be the new home page
* The nav will only show the home link for those with permission

### CODE REVIEW

**Comment by dorianjp** [comment]
[Portal navigation after log-in](https://trello.com/c/gPhP8j6T/800-portal-navigation-after-log-in)

File: covid_key/templates/covid_key/landing.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  <h1>{% trans "Welcome to the COVID Alert portal" %}</h1>
+
+  <div class="box-container">
+    <a href="{% url 'otk_start' %}">
+      <div class="box-button">
+        <div class="box-button-column">
+          <div class="box-button-row">
+            <div>
+              <b>{% trans 'Give individuals a one-time key' %}</b>
+            </div>
```
Comment (dsamojlenko) [inline covid_key/templates/covid_key/landing.html:16]: I wonder if semantically these should be headings? Probably `<h2>` given the page context

File: covid_key/tests.py [hunk 2]
```diff
@@ -26,13 +27,32 @@ def test_apps(self):
...
+        self.login()
+        response = self.client.get(reverse("start"))
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, "<h1>Welcome to the COVID Alert portal</h1>")
+
+    def test_landing_view_without_alert_perm(self):
+        self.login(self.user2_credentials)
+        response = self.client.get(reverse("start"))
+        self.assertEqual(response.status_code, 302)
+        self.assertEqual(response.url, reverse("otk_start"))
```
Comment (dsamojlenko) [inline covid_key/tests.py:46]: yay tests!

File: locale/fr/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -1291,6 +1338,14 @@ msgstr "Choix de langue"
 msgid "Product navigation"
 msgstr "Navigation du produit"
 
+#: portal/templates/includes/nav_main.html
+#: register/templates/register/contact_us.html
+#: register/templates/register/help_page.html
+#: register/templates/register/privacy.html
+#: register/templates/register/terms.html
+msgid "Home"
+msgstr ""
```
Comment (dsamojlenko) [inline locale/fr/LC_MESSAGES/django.po:1347]: Probably "Accueil"

**Comment by dsamojlenko** [comment]
Nice work ... scales down nicely. Just the one note on semantics, and the Home link translation.

File: covid_key/templates/covid_key/landing.html (hunk 1 above)
Comment (dorianjp) [reply covid_key/templates/covid_key/landing.html:16]: sure

**Review by dsamojlenko — APPROVED** [review]

**Pull request merged**